src/query/interpreter.cpp

```diff
@@ -2922,6 +2922,31 @@ PreparedQuery PrepareInfoQuery(ParsedQuery parsed_query, bool in_explicit_transa
           results.push_back({TypedValue("label+property"), TypedValue(db->LabelToName(item.first)),
                              TypedValue(db->PropertyToName(item.second))});
         }
+
+        std::sort(results.begin(), results.end(), [](const auto record_1, const auto record_2) {
+          const auto type_1 = record_1[0].ValueString();
+          const auto type_2 = record_2[0].ValueString();
+
+          if (type_1 != type_2) {
+            return type_1 < type_2;
+          }
+
+          if (type_1 == "label") {
```

Comment:
"label" should be some constant, I think I added it somewhere in the disk storage, try to find it and reuse, if not add your own.